app/workers/downstream_publish_worker.rb

```diff
@@ -0,0 +1,93 @@
+class DownstreamPublishWorker
+  attr_reader :content_item_id, :payload_version, :message_queue_update_type, :update_dependencies
+  include Sidekiq::Worker
+  include PerformAsyncInQueue
+
+  HIGH_QUEUE = "downstream_high".freeze
+  LOW_QUEUE = "downstream_low".freeze
+
+  sidekiq_options queue: HIGH_QUEUE
+
+  def perform(args = {})
+    assign_attributes(args.symbolize_keys)
+
+    unless web_content_item
+      raise CommandError.new(
+        code: 404,
+        message: "The content item for id: #{content_item_id} was not found",
+      )
```

Comment:
Valid point - using it here is consistent with how existing workers use it: https://github.com/alphagov/publishing-api/blob/master/app/workers/presented_content_store_worker.rb#L33 so may need to get some further input on this on the next on whether or how this should error.